eland/tests/ml/test_imported_ml_model_pytest.py

```diff
@@ -62,6 +62,10 @@
 )
 
 
+def random_rows(data, size):
```

Comment:
this provides MUCH better testing. 

The training data is random, and selecting a random subset to infer against provides much better coverage than a static two rows.